spec_creation/Validate_spec_before_upload.ipynb

```diff
@@ -233,24 +211,21 @@
     "                        lonlat_swap(wpc), popup=\"%d\" % i,\n",
     "                        icon=fof.DivIcon(class_name='leaflet-div-icon')).add_to(curr_map)\n",
     "    \n",
+    "    # iterate over all reroutes\n",
     "    for rc in trip[\"route_coords\"]:\n",
-    "        print(\"Found %d coordinates for the route\" % (len(rc[\"geometry\"][\"coordinates\"][0])))\n",
-    "    \n",
-    "    latlng_routes = [[lonlat_swap(rc) for rc in r[\"geometry\"][\"coordinates\"][0]] for r in trip[\"route_coords\"]]\n",
```

Comment:
where are you doing `lonlat_swap` in the modified code?

Reply:
seems like you are just using the coords directly?

Reply:
for some reason the maps only showed up correctly when I didn't coordinate swap. However, this may be the reason why some of my routes are off. working on fixing this right now.